src/main/java/seedu/module/model/ModuleManager.java

```diff
@@ -0,0 +1,102 @@
+package seedu.module.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import seedu.module.model.task.Module;
+import seedu.module.model.task.Task;
+
+/**
+ * Represents a collection of Modules, and the list of Tasks associated with each.
+ */
+public class ModuleManager {
+    private static HashMap<String, List<Task>> mapping;
+    private static List<String> existingModulesInStr;
```

Comment:
Here existingModuleInStr may be a bit confused since I think it means the list of valid module code right? existingModuleInStr seems a bit closer to the modules of current mapping though it's just personal consideration.